GNUmakefile

```diff
@@ -149,19 +149,14 @@ $(PACKAGE)-suse.spec: $(PACKAGE).spec.in GNUmakefile
 	sed -i s:corosynclib:libcorosync:g $@
 	sed -i s:libexecdir}/lcrso:libdir}/lcrso:g $@
 	sed -i 's:%{name}-libs:lib%{name}3:g' $@
-	sed -i s:heartbeat-libs:heartbeat:g $@
 	sed -i s:cluster-glue-libs:libglue:g $@
-	sed -i s:libselinux-devel:automake:g $@
 	sed -i s:lm_sensors-devel:automake:g $@
 	sed -i s:bzip2-devel:libbz2-devel:g $@
 	sed -i s:bcond_without\ publican:bcond_with\ publican:g $@
 	sed -i s:docbook-style-xsl:docbook-xsl-stylesheets:g $@
 	sed -i s:libtool-ltdl-devel::g $@
-	sed -i s:dbus-devel:dbus-1-devel:g $@
 	sed -i s:publican::g $@
 	sed -i s:byacc::g $@
-	sed -i s:global\ cs_major.*:global\ cs_major\ 1:g $@
-	sed -i s:global\ cs_minor.*:global\ cs_minor\ 4:g $@
```

Comment:
BTW I'm not trying to get this target fully working at this point, just removing patterns that no longer exist in the original spec. I'll revisit this target later after researching a bit.